RadialMenuControl/UserControl/RadialMenu.xaml.cs

```diff
@@ -187,16 +195,87 @@ private void _clearPie(bool storePrevious)
         /// <param name="storePrevious">Should we store the previous pie (for back navigation)?</param>
         public void ChangeToCustomMenu(object s, MenuBase newSubMenu, bool storePrevious)
         {
-            _clearPie(storePrevious);
-            // Redraw
-            Pie.Draw();
-            Pie.UpdateLayout();
-            // TODO use just an auxilary canvas and add custom controls to that
-            newSubMenu.Diameter = Diameter;
-            CustomRadialControlRoot.Children.Add(newSubMenu);
-            newSubMenu.UpdateLayout();
+           
+            if(newSubMenu is ListSubMenu)

```

Comment:
The idea of having a canvas, `CustomRadialControlRoot` is so that you can add any MenuBase menu to it. RadialMenu shouldn't have to know about any deatails of the custom submenu. This allows us to add as many types of menus and menus outside of our package can also be loaded, so long they subclass from `MenuBase`.